codecov.yml

```diff
@@ -10,3 +10,8 @@ coverage:
       # TODO: Consider relaxing the patch target vs range for the global project in the long term, being more lenient on the patches themselves.
       default:
         threshold: 1%
+
+parsers:
+  cobertura:
+    partials_as_hits: false
+    handle_missing_conditons: false
```

Comment:
Visiting other projects I came to the conclusion that's an inherent limitation of the tools in use. Let's keep consistent with what others are doing and let be it 🤷🏽‍♂️

Reply:
Yeah, it's unfortunate that it seems to not do what it implies it should be doing, but I think as long as we have a standard it's ok.

Reply:
(I removed the commit that added this change, by the way)